includes/multi-currency/client/index.js

```diff
@@ -10,38 +10,22 @@ import ReactDOM from 'react-dom';
 import MultiCurrencySettings from './settings/multi-currency';
 import SingleCurrencySettings from './settings/single-currency';
 import MultiCurrencySettingsContext from './context';
+import UnbundledWpComponentsProvider from 'wcpay/wordpress-components-context/unbundled-wp-components-provider';
 
 const MultiCurrencySettingsPage = () => {
-	const [
```

Comment:
The previous implementation was using two `useState`: one for `isSingleCurrencyScreenOpen`, and another for `currencyCodeToShowSettingsFor`.

But we can only use `currencyCodeToShowSettingsFor`, `isSingleCurrencyScreenOpen` is basically a "falsy" version of `currencyCodeToShowSettingsFor`.
Having two state values where one can be inferred from the other seems wasteful, and can create sync issues.